infra/chromadb_compaction/utils/metrics.py

```
@@ -0,0 +1,261 @@
+"""CloudWatch custom metrics for ChromaDB compaction operations."""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)